src/content/docs/query-your-data/nrql-new-relic-query-language/get-started/introduction-nrql-new-relics-query-language.mdx

```diff
@@ -102,65 +102,113 @@ import queriesnrqlEventDefinitionsQueryBuilder from 'images/queries-nrql_screens
 
 import queriesnrqlAttributeDefinitionsQueryBuilder from 'images/queries-nrql_screenshot-crop_attribute-definitions-query-builder.png'
 
-One way to [query your New Relic data](/docs/using-new-relic/data/understand-data/query-new-relic-data) is with the New Relic Query Language (NRQL). This resource explains what NRQL is, when and how you can use it, and basic syntax rules. For more detailed information on querying, including a listing of clauses and functions and example queries, see [NRQL syntax, clauses, and functions](/docs/query-your-data/nrql-new-relic-query-language/get-started/nrql-syntax-clauses-functions).
+import queriesnrqlconsole from 'images/queriesNrql_screenshot-full_nrql-console.png'
 
-We've included a NRQL Lessons application on our [opensource.newrelic.com](https://opensource.newrelic.com/projects/newrelic/nr1-learn-nrql) site to help you quickly see NRQL's value and power by using your own data. To learn how to install and use the NRQL Lessons app, watch this short YouTube video (approx. 2:45 minutes).
+import nr1NrqlAlertConditions from 'images/nr1_nrql_alert_conditions.png'
+
+import NerdGraphscreenshot from 'images/apis-and-data_screenshot-crop_NerdGraph-internal-docs.png'
+
+The New Relic Query Language (NRQL) is one of the most versatile and powerful tools you can use to [query your data](/docs/using-new-relic/data/understand-data/query-new-relic-data), but it can seem overwhelming at first glance. Don't worry! Here is some information to give you a good foundational understanding of NRQL, including what it is, how to use it, and some tips and tricks that will help you get the most out of your queries. 
```

Comment:
I kind of want a value prop here-- like why would someone want to query their data? What benefit would querying their data provide? Just something to say--"I know this is scary but you could WIN big!" (obviously less cheesy than that!)

Reply:
Agreed, I added something that will hopefully do just that.